src/components/editor/ImportResourceTemplate.jsx

```diff
@@ -85,20 +85,44 @@ class ImportResourceTemplate extends Component {
   updateStateFromServerResponse = (response, profileCount, updateKey) => {
     // HTTP status 409 == Conflict
     const showModal = response.status === 409 && this.state.createResourceError.length >= profileCount
-
-    const location = response.headers.location || ''
-    const newMessage = `${this.humanReadableStatus(response.status)} ${location}`
-    const newState = {
-      message: [...this.state.message, newMessage],
-      updateKey: updateKey
-    }
+    const newMessage = `${this.humanReadableStatus(response.status)} ${this.humanReadableLocation(response)}`
+    const newState = { updateKey: updateKey }
 
     if (showModal)
       newState.modalShow = true
 
+    // `response.ok` is a Boolean that returns true if the HTTP status is in the
+    // 200-299 range, indicating the HTTP request was successful
+    if (response.ok) {
+      // Wipe out past messages and errors if response was successful
+      newState.createResourceError = []
+      newState.message = [newMessage]
+    } else {
+      // Response not OK, so append message to list of current messages
+      newState.message = [...this.state.message, newMessage]
```

Comment:
Like this use of the spread operator here and the appending of the `newMessage` to the list